tests/functional/vault/test_misc.py

```diff
@@ -124,14 +124,38 @@ def test_negative_available_deposit_limit(Vault, token, gov):
 
 
 def test_sweep(gov, vault, rando, token, other_token):
-    token.transfer(vault, token.balanceOf(gov), {"from": gov})
-    other_token.transfer(vault, other_token.balanceOf(gov), {"from": gov})
+    # Vault wrapped token reverts on 0 sweep
+    with brownie.reverts():
+        vault.sweep(token, {"from": gov})
 
-    # Vault wrapped token doesn't work
+    # Airdrop want token
+    airdropAmount = token.balanceOf(gov) // 2
+    token.transfer(vault, airdropAmount, {"from": gov})
```

Comment:
we can add a test in deposit test cases for a vault with totalsupply = 0 + airdrop manipulation + first depositor can't benefit from shares to be sure we cover that edge case

Reply:
done in https://github.com/yearn/yearn-vaults/commit/ba735e9b44634cb6da32fafb64fef4d4b1c379cb